src/components/KSelect/KSelectItem.vue

```diff
@@ -10,7 +10,8 @@
       class="d-block"
     >
       <button
-        :class="{ disabled, selected: item.selected }"
+        :class="{ selected: item.selected }"
+        :disabled="item.disabled"
```

Comment:
I think we need to update this logic a bit so that we don't end up with `disabled="false"` on the element.
```suggestion
        :disabled="item.disabled === true ? true : undefined"
```